server/neptune/workflows/activities/conftest/summary.go

```diff
@@ -0,0 +1,56 @@
+package conftest
+
+import (
+	"fmt"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/activities"
+	"strings"
+)
+
+type ValidateSummary struct {
+	Failures  []string
+	Successes []string
+}
+
+func NewValidateSummaryFromResults(results []activities.ValidationResult) ValidateSummary {
+	if len(results) == 0 {
+		return ValidateSummary{}
+	}
+
+	var failures []string
+	var successes []string
+	for _, result := range results {
+		summary := result.PolicySet.Name
+		switch result.Status {
```

Comment:
is there a default case we should handle?

Reply:
oops nope, I set it up as a switch because we may one day want to support another case (when conftest returns Warn instead of Fail). I can add an error log in the default since we shouldn't hit that

Reply:
actually, I just went ahead and use the if-else, no reason to support that now since we have no plans to use Warn at the moment.